src/Microsoft.Sbom.Api/SBOMValidator.cs

```diff
@@ -23,16 +26,25 @@ public class SbomValidator : ISBOMValidator
     private readonly IWorkflow<SbomParserBasedValidationWorkflow> sbomParserBasedValidationWorkflow;
     private readonly IRecorder recorder;
     private readonly IEnumerable<ConfigValidator> configValidators;
+    private readonly IConfiguration configuration;
+    private readonly ISbomConfigProvider sbomConfigs;
+    private readonly IFileSystemUtils fileSystemUtils;
 
     public SbomValidator(
         IWorkflow<SbomParserBasedValidationWorkflow> sbomParserBasedValidationWorkflow,
         IRecorder recorder,
         IEnumerable<ConfigValidator> configValidators,
-        ConfigSanitizer configSanitizer)
+        ConfigSanitizer configSanitizer,
+        IConfiguration configuration,
+        ISbomConfigProvider sbomConfigs,
```

Comment:
This is a breaking change, please add a new constructor that takes the new parameters